compiler/src/iree/compiler/API/Internal/CMakeLists.txt

```diff
@@ -113,6 +113,7 @@ iree_cc_library(
     "IREEGPUDialectCAPI.cpp"
   DEPS
     IREELLVMIncludeSetup
+    LLVMSupport
```

Comment:
just curious: why suddenly this? I only ask because I'm just this moment figuring out (elsewhere) whether to package LLVMSupport or not.

Reply:
Oh, I need to clean this up. Thanks.